src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetFullPathNameW.cs

```diff
@@ -8,16 +8,13 @@ internal static partial class Interop
 {
     internal static partial class Kernel32
     {
+#pragma warning disable DLLIMPORTGENANALYZER015 // Use 'GeneratedDllImportAttribute' instead of 'DllImportAttribute' to generate P/Invoke marshalling code at compile time
+                                                // Disabled since GetFullPathNameW takes 'ref's into buffers (the ref parameters don't represent single characters).
```

Comment:
What is going to be the recommended way to make these work with GeneratedDllImports?

Reply:
This particular P/Invoke took a dependency on the fact that chars are conditionally blittable, so no copying was done. So far, the Interop team has decided to not consider `char`s as conditionally blittable.

To make this particular P/Invoke work, there are a few different options:

1. Update the generator to consider `char` parameters as blittable when `CharSet=CharSet.Unicode`.
2. Update this P/Invoke to take `char*` parameters instead of `ref char` and pass in pinned refs.
3. Update the first parameter to take a `string` (which will be pinned) or a `ReadOnlySpan<char>` (using the new Span marshalers) and the second to take a `char*`, which will be passed without change. As the usage of this method calculates the refs to pass in, from a `ReadOnlySpan<char>` and a `ValueStringBuilder`, this would be possible without too much trouble.

Reply:
With the merge conflicts I had after the property order normalization, I just fixed this as part of resolving those.